docs/site/Creating-components.md

```diff
@@ -6,7 +6,7 @@ sidebar: lb4_sidebar
 permalink: /doc/en/lb4/Creating-components.html
 ---
 
-As explained in [Using Components](Using-components.md), a typical LoopBack
+As explained in [Using Components](components.md), a typical LoopBack
```

Comment:
I believe you're referring to https://github.com/strongloop/loopback-next/blob/master/docs/site/Components.md. The file name here is case-sensitive. 

```suggestion
As explained in [Using Components](Components.md), a typical LoopBack
```

Reply:
I probably am, yes. 
It was just a guess to where the docs were supposed to be taking me.
If that's the correct place, then I will change it.

Reply:
If that's the case, yes, please use `Components.md` instead. Thanks.